### DISCUSSION TITLE
Submitting a feature request

### DISCUSSION THREAD

**Discussion by WarpRules** [google/googletest#3425] [Community Help]
I see that some entries in the issues section are labeled with the "enhancement" label, I assume meaning a feature request. I have an idea for an enhancement, but for the life of me I cannot figure out how to open a new "enhancement" type issue. The only thing that seems to be available is "bug report", with a "bug" label which cannot be changed (or at least I haven't figured out a way to change it). Is this even possible?

**Comment by nicola-lunghi** [comment]
same here

**Comment by horiyu** [comment]
I'm going through something similar, but I have a sense that this issue has been resolved.

[Fix indentation syntax error in feature request issue template](https://github.com/google/googletest/commit/e1ee0fa3e121e819c7a76d4d3b417de8f73ebfd7) https://github.com/google/googletest/commit/e1ee0fa3e121e819c7a76d4d3b417de8f73ebfd7
